PUBLISHING.md

```diff
@@ -1,4 +1,11 @@
-# Crates publishing
+# Publishing
+
+## OCaml
+
+There is only the package `hax-engine`, that includes a binary and a
```

Comment:
How does publishing work here? Also, did you want to publish this to opam? I don't see it there.

Reply:
I pushed a note, where I explain what we discussed (pinning git basically)

Reply:
If the note looks good to you, let's resolve this discussion and merge the PR

Reply:
lgtm
Please add a note how to update the README if needed, or how the release tag get's on there. Since we don't push it anywhere else, that's essentially the release then and we should describe here how to do that.
Then you can resolve this and merge.

Reply:
makes sense, started something in branch `document-stable-install`